android-sdk/src/main/java/co/elastic/apm/android/sdk/traces/common/attributes/SdkIdVisitor.java

```diff
@@ -29,6 +29,7 @@ public class SdkIdVisitor implements AttributesBuilderVisitor {
     public void visit(AttributesBuilder builder) {
         builder.put(ResourceAttributes.TELEMETRY_SDK_NAME, "android")
                 .put(ResourceAttributes.TELEMETRY_SDK_VERSION, BuildConfig.APM_AGENT_VERSION)
+                .put("type", "mobile")
```

Comment:
When is this `visit(...)`invoked? Once to setup the resource attributes or for each span?

I'm asking because, `TELEMETRY_SDK_NAME`, `TELEMETRY_SDK_VERSION` and `TELEMETRY_SDK_LANGUAGE` are resource attributes (so they have a device / process-wide scope). While `type -> mobile` should be a span attribute.

So, if this visit(...) method is used to build resource attributes, then setting the type `mobile` here is wrong. We should rather do it, when we start a UI root span.

Reply:
Oh I see, yes it's currently as you say, being set as a resource attr. I'll make the changes